apps/zotonic_core/include/zotonic_notifications.hrl

```diff
@@ -1188,6 +1188,14 @@
 }).
 
 
+%% @doc Delegates the request processing.
+%% Type: foldl
+%% Return: modified context
+-record(middleware, {
+    on :: executed | welformed | handled
```

Comment:
Maybe `executed` should be called `init` or `start` ?
That might be clearer.

Reply:
I'm in doubt about what could be better, but I think `start` would be best. What do you think?

Reply:
Or maybe `request`?

Reply:
`request` sounds good to me.  @mmzeeman @vkatsuba  ok?

Reply:
Sounds good.

Reply:
Changed `executed` to `request`.